gpu/types.go

```diff
@@ -38,6 +38,30 @@ type GpuInfo struct {
 	// TODO other performance capability info to help in scheduling decisions
 }
 
+type CPUInfo struct {
+	GpuInfo
+}
+
+type CudaGPUInfo struct {
+	GpuInfo
+	index int // nolint: unused
+}
+type CudaGPUInfoList []CudaGPUInfo
+
+type RocmGPUInfo struct {
+	GpuInfo
+	usedFilepath string // nolint: unused
```

Comment:
```suggestion
	usedFilepath string
```

I believe this is used now below

Reply:
I believe the nolint stuff is still needed since this types.go is on all 3 platforms (mac, windows, linux)

Reply:
Ah, figured that's why they might be there 👍